src/libsync/bandwidthmanager.cpp

```diff
@@ -375,18 +377,18 @@ void BandwidthManager::switchingTimerExpired()
 
 void BandwidthManager::absoluteLimitTimerExpired()
 {
-    if (usingAbsoluteUploadLimit() && _absoluteUploadDeviceList.count() > 0) {
-        qint64 quotaPerDevice = _currentUploadLimit / qMax(1, _absoluteUploadDeviceList.count());
-        qCDebug(lcBandwidthManager) << quotaPerDevice << _absoluteUploadDeviceList.count() << _currentUploadLimit;
+    if (usingAbsoluteUploadLimit() && !_absoluteUploadDeviceList.empty()) {
+        qint64 quotaPerDevice = _currentUploadLimit / qMax(size_t(1), _absoluteUploadDeviceList.size());
+        qCDebug(lcBandwidthManager) << quotaPerDevice << _absoluteUploadDeviceList.size() << _currentUploadLimit;
         for (auto *device : qAsConst(_absoluteUploadDeviceList)) {
             device->giveBandwidthQuota(quotaPerDevice);
             qCDebug(lcBandwidthManager) << "Gave " << quotaPerDevice / 1024.0 << " kB to" << device;
         }
     }
-    if (usingAbsoluteDownloadLimit() && _downloadJobList.count() > 0) {
-        qint64 quotaPerJob = _currentDownloadLimit / qMax(1, _downloadJobList.count());
-        qCDebug(lcBandwidthManager) << quotaPerJob << _downloadJobList.count() << _currentDownloadLimit;
-        for (auto *j : qAsConst(_downloadJobList)) {
+    if (usingAbsoluteDownloadLimit() && !_downloadJobList.empty()) {
+        qint64 quotaPerJob = _currentDownloadLimit / qMax(size_t(1), _downloadJobList.size());
```

Comment:
same

Reply:
Yes, same here: why use `qMax` when you know the list is not empty?